packages/react-kit/src/components/form/Upload/BaseUpload.tsx

```diff
@@ -91,6 +93,9 @@ function BaseUpload({
   const isVideoOnly = mimetypes.every((mimetype) =>
     mimetype.startsWith("video/")
   );
+  const isFileOnly = mimetypes.every((mimetype) =>
+    mimetype.startsWith("application/")
```

Comment:
```suggestion
    mimetype.startsWith("application/pdf")
```

Reply:
We do not care about docx / doc etc?

Reply:
that's a question for Sukrita, but if you can then also upload a .js file and show the pdf logo it looks weird, if we only plan to use pdf then let's not overcomplicate I guess?

Reply:
Totally agree with you, i change all. No time to think about it

Reply:
then let's make this change @AdrianBielecAriane

Reply:
Done